news/services/metadata_extractor.py

```
@@ -0,0 +1,343 @@
+            domain = urlparse(url).netloc.lower()
+
+            # 정확한 매칭
+            if domain in self.PRESS_DOMAIN_MAP:
+                return self.PRESS_DOMAIN_MAP[domain]
+
+            # 서브도메인 매칭
+            for key, value in self.PRESS_DOMAIN_MAP.items():
+                if value and domain.endswith(key):
+                    return value
+        except Exception:
+            pass
```

Ruff (0.14.11)

`try`-`except`-`pass` detected, consider logging the exception

(S110)